newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -94,48 +90,73 @@ export default class CommentEvent extends React.Component<
   };
 
   render() {
-    const commentEvent = gd.asCommentEvent(this.props.event);
-    const color = rgbToHex(
-      commentEvent.getBackgroundColorRed(),
-      commentEvent.getBackgroundColorGreen(),
-      commentEvent.getBackgroundColorBlue()
-    );
-    const textColor = rgbToHex(
-      commentEvent.getTextColorRed(),
-      commentEvent.getTextColorGreen(),
-      commentEvent.getTextColorBlue()
-    );
+    var commentEvent = gd.asCommentEvent(this.props.event);
+
+    const r = commentEvent.getBackgroundColorRed(),
+      g = commentEvent.getBackgroundColorGreen(),
+      b = commentEvent.getBackgroundColorBlue();
+
+    const textColor = (r + g + b) / 3 > 200 ? 'black' : 'black';
```

Comment:
One should be white no?

Reply:
I use this trick actually comments are yellow, and white on yellow i see nothing.
I will use a other way to do.
Fixed